README.md

```diff
@@ -48,6 +48,44 @@ Well, it makes sense as a starting point. But there are several possible drawbac
 </details>
 
 
+<details>
+<summary>What is the difference between Open Metric Learning and PyTorch Metric Learning?</summary>
+<p>
+
+[PML](https://github.com/KevinMusgrave/pytorch-metric-learning) is the popular library for Metric Learning
+and it includes rich collection of losses, miners, distances, and reducers; that is why we provide straightforward
+[examples](https://github.com/OML-Team/open-metric-learning#usage-with-pytorch-metric-learning) of using them with OML.
+Initially, we tried to use PML, but in the end, we came up with our library, which is more pipeline / recipes oriented.
```

Comment:
Probably we need some sentence before list of advantages, like: "Here's what OML is capable of:" or "That is how OML differs from PML:" or something else.

Reply:
thx